### PR TITLE
Feature: Minor improvements to InfluxHistorian

### PR DESCRIPTION
**Pull request by WhoBrokeTheBuild**
* Start time query now uses >= instead of >
* fVal/iVal/bVal is now discovered at runtime, instead of specifying in the `SELECT` node

### CODE REVIEW

**Review by joshStillerman — APPROVED** [review]

**Review by santorofer — APPROVED** [review]

**Comment by joshStillerman** [comment]
retest this please

File: pydevices/HtsDevices/influxhistorian.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ class INFLUXHISTORIAN(MDSplus.Device):
         parts.append({'path': name + ":WHERE",     'type': 'text',
                       'options': ('no_write_shot',)})
         parts.append({'path': name + ":SELECT",    'type': 'text',
-                      'options': ('no_write_shot',)})
+                      'options': ('no_write_shot',)}) # No longer used
```
Comment (zack-vii) [inline pydevices/HtsDevices/influxhistorian.py:35]: this is bad and the reason why i would not release halve baked devices

**Comment by zack-vii** [comment]
test this please

**Comment by zack-vii** [comment]
test fails due to unrelated issue.. can rebase as fork it is not on MDSplus

**Pull request merged**